manifests/implementation/capactio/capact/scream.yaml

```diff
@@ -0,0 +1,49 @@
+ocfVersion: 0.0.1
+revision: 0.1.0
+kind: Implementation
+metadata:
+  prefix: cap.implementation.capactio.capact
+  name: scream
+  displayName: "Scream CAPACT"
+  description: "Capact SCREAM Action"
+  documentationURL: https://capact.dev
+  supportURL: https://capact.dev
+  license:
+    name: "Apache 2.0"
+  maintainers:
+    - email: team-dev@capact.dev
+      name: Capact Dev Team
+      url: https://capact.dev
+
```

Comment:
I think it would be great to pick Implementations based on Attributes / requires for the hello-world example. Maybe Implementation constraints with Attribute would be easier to understand. A policy with exact Implementation path doesn't look nice as it needs to be known by the user.